suite-common/suite-types/src/device.ts

```diff
@@ -40,7 +40,7 @@ export interface ExtendedDevice {
     ts: number;
     buttonRequests: ButtonRequest[];
     metadata: DeviceMetadata;
-    walletNumber?: number; // number of hidden wallet intended to be used in UI
```

Comment:
here from `hidden` to `passphrase`, but above you did from `passphrase` to `hidden`? why what is correct naming now?

Reply:
Here for example: https://github.com/trezor/trezor-suite/pull/13059/files#diff-2db736617f53aec5e3acbfeddb95cf37f79cf3bbd8f217da1d81d91fff13bf16L99

Reply:
That is not true 🤷‍♂️😀 it should be hidden -> passphrase everywhere, because hidden is deprecated now

Reply:
yea... sorry :D